### PR TITLE
Complete virology addition

### PR DESCRIPTION
**Pull request by Kaksisilma**
## About The Pull Request

image

Re-adds virology, along with skill perks, machinery and et-cetera.

This PR is dependent on https://github.com/BoHBranch/BoH-Bay/pull/1432, and it must be merged first for it to work.

Still heavily WIP, pending mapping changes and testing.

## Why It's Good For The Game

Virology is an intricate and !!!FUN!!! system, and I would say that ss13 is not complete without it. Also Carl wants it.

## Did You Test It?

Yes. Everything is functional.
I have not tested Lar-Maria, though.

## Authorship

Kaksisilma

## Changelog

:cl:
rscadd: Virology has been re-added.
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -669,6 +669,36 @@ var/list/admin_verbs_mod = list(
 	log_and_message_admins("created an admin explosion at [epicenter.loc].")
 	SSstatistics.add_field_details("admin_verb","DB") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/give_disease2(mob/T as mob in SSmobs.mob_list) // -- Giacom
```
Comment (Kaksisilma) [inline code/modules/admin/admin_verbs.dm:672]: This give disease proc is not working properly and does not even appear in the Fun tab. I'm not sure why.

Comment (ProbablyCarl) [reply code/modules/admin/admin_verbs.dm:672]: Shit's fucked. It has been for a hot minute - don't worry too heavily about it.

**Comment by Cupax3** [comment]
Okay this is definitely going to need to be squashed before merge

**Comment by Kaksisilma** [comment]
> Okay this is definitely going to need to be squashed before merge

Done

**Pull request merged**